api/util/CreateUserScript.js

```diff
@@ -0,0 +1,125 @@
+'use strict';
+const User = require('../models/User.js');
+const inquirer = require('inquirer');
+const mongoose = require('mongoose');
+mongoose.Promise = require('bluebird');
+let membershipState = require('../../src/Enums.js');
+membershipState = membershipState.membershipState;
+
+const redColor = '\x1b[31m';
+const greenColor = '\x1b[32m';
+const blueColor = '\x1b[34m';
+const defaultColor = '\x1b[0m';
+
+// eslint-disable-next-line no-console
+console.log(blueColor, 'Welcome to the SCE User Creation CLI!', defaultColor);
+inquirer
+  .prompt([
+    {
+      type: 'input',
+      name: 'email',
+      message: 'What\'s your email:'
+    },
+    {
+      type: 'input',
+      name: 'firstName',
+      message: 'Please enter a first name for your account (default: "Dummy"):'
+    },
+    {
+      type: 'input',
+      name: 'lastName',
+      message: 'Please enter a last name for your account (defualt: "Account"):'
+    },
+    {
+      type: 'input',
+      name: 'password',
+      message: 'Please enter a password (default: "sce"):'
+    },
+    {
+      type: 'list',
+      name: 'accountLevel',
+      message: 'Choose an account level from the list below: ',
+      choices: [
+        {
+          name: 'Admin',
+          value: membershipState.ADMIN
+        },
+        {
+          name: 'Officer',
+          value: membershipState.OFFICER
+        },
+        {
+          name: 'Member',
+          value: membershipState.MEMBER
+        },
+        {
+          name: 'Non-Member',
+          value: membershipState.NON_MEMBER
+        },
+        {
+          name: 'Pending',
+          value: membershipState.PENDING
+        },
+        {
+          name: 'Banned',
+          value: membershipState.BANNED
+        }
+      ],
+      filter: function(val) {
+        // eslint-disable-next-line no-console
+        console.log(val);
```

Comment:
change every instance of console.log in this file to console.debug

the reason we do this is because:
1. we know people use console.log for printing and we want to discourage leaving print statements in code
2. for the print statements we do feel need to stay, we use console.debug to avoid limiting errors